tests/CookieTest.php

```diff
@@ -19,62 +20,79 @@
  */
 final class CookieTest extends TestCase
 {
-    public function testDefaults(): void
+	/** @dataProvider \Ergebnis\DataProvider\StringProvider::arbitrary() */
+    public function testDefaults(string $name): void
```

Comment:
Your test uses only one single name again and again and again. This would not realy meaningful because it could mean that class is only made for this input and everything other possible inputs are not tested.

For this the DataProvider Package from @Localheinz should help :) https://github.com/ergebnis/data-provider